news-bundle/src/EventListener/InsertTagsListener.php

```diff
@@ -55,13 +56,17 @@ private function replaceNewsFeedInsertTag(string $feedId): string
     {
         $this->framework->initialize();
 
-        $adapter = $this->framework->getAdapter(NewsFeedModel::class);
```

Comment:
I'm not sure we should actually keep the `news_feed` insert tag. It will not correctly work after an update, and it's the same as `link_url` anyway. Removing it (and maybe writing something to the logs) would probably be easier to spot errors.

Reply:
Yes, good point with `link_url` I'll trigger a `warning` in the logs.

Reply:
See 04a5351ae